_news-items/2025-05-19-unsw.html

```diff
@@ -0,0 +1,40 @@
+---
+# Copyright 2025 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Welcome UNSW, Bronze sponsor of the seL4 Summit 2025
```

Comment:
I would change the titles for sponsorship to "Thank you UNSW, Bronze sponsor..." rather than "Welcome..."